src/main/java/io/glutenproject/integration/tpc/Tpc.java

```diff
@@ -90,10 +97,24 @@ private SparkConf pickSparkConf(String backendType) {
     return conf;
   }
 
+  private void setUserConf(SparkConf conf) {
+    if (sparkConf.isEmpty()) {
+      return;
+    }
+    Arrays.stream(sparkConf.split(";")).forEach(c -> {
+      int idx = c.indexOf("=");
+      if (idx == 0) {
+        throw new IllegalArgumentException("Conf format should be a=b");
+      }
+      conf.set(c.substring(0, idx), c.substring(idx + 1));
+    });
+  }
+
   @Override
   public Integer call() throws Exception {
     final SparkConf baselineConf = pickSparkConf(baselineBackendType);
     final SparkConf testConf = pickSparkConf(backendType);
+    setUserConf(testConf);
```

Comment:
Looks that if an custom option can be set, it's better to set it to both `baselineConf` and `testConf`, or it might become a debatable design since a customized option will easily make the `baseline` vs `test` comparison unfair.

Can you show a real world case about setting a custom option only to `testConf`?

Reply:
Because in my case, I always set gluten own `GlutenConfig`, so I only set it to testConf. In most cases, user specify gluten config in baseLine makes no side effect, so I will change it both `baseLine` and `test`